Package.swift

```diff
@@ -0,0 +1,32 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+import PackageDescription
+
+let package = Package(
+    name: "JitsiMeetSDK",
+    platforms: [.iOS(.v14)],
```

Comment:
The minimum is 13.4

Reply:
Unfortunately we don't have this option with SPM, it's either 13 or 14.

Reply:
Oh, ok then.